python/tvm/relay/frontend/onnx.py

```diff
@@ -2263,6 +2265,14 @@ def from_onnx(model,
     dtype : str or dict of str to str
         The input types to the graph
 
+    big_model : bool
+        Whether the size of the input model is larger than 2GB
+
+    model_path : str, optional
+        The path of input model, and the external data
+        files need under the same directory
+        Must be set if 'big_model' is True
```

Comment:
In this case, I think it's better to check whether model_path is None, instead of providing an additional big_model argument.

Reply:
@liangfu Thanks for your advice.
As I commented on @adityaatluri 's advice, I think it is better to introduce two arguments, 'model_path' and 'external_data_path,' to replace 'model,' 'big_model,' 'model_path.' in my commit.
The newest ONNX API can load a model via model path and external data files' path (external files' path is needed if they are not under the same directory with the model file) without any other arguments.